lib/express-rate-limit.js

```diff
@@ -85,6 +86,18 @@ function RateLimit(options) {
               );
             }
           }
+          if (options.draft_polli_ratelimit_headers) {
+            res.setHeader("RateLimit-Limit", max);
+            res.setHeader("RateLimit-Remaining", req.rateLimit.remaining);
+            if (resetTime instanceof Date) {
+              // if we have a resetTime, also provide the current date to help avoid issues with incorrect clocks
+              res.setHeader("Date", new Date().toGMTString());
+              res.setHeader(
+                "RateLimit-Reset",
+                Math.ceil((resetTime.getTime() - Date.now()) / 1000)
```

Comment:
I'd ensure that returns a non-negative number.

Reply:
Sure, Math.abs() would return the absolute value. I'll do that.

Reply:
I think something like 

```
resetTime.getTime() - Date.now() < 0 ? 0 : Math.ceil(resetTime.getTime() - Date.now()/1000) 
```

Reply:
That almost works, but it lost the parens around `resetTime.getTime() - Date.now()`. This gets the correct result with a little less code:

```js
Math.max(0, Math.ceil((resetTime.getTime() - Date.now())/1000));
```

or, to improve readability:

```js
var deltaSeconds =  Math.ceil((resetTime.getTime() - Date.now()) / 1000);
res.setHeader("RateLimit-Reset", Math.max(0, deltaSeconds));
```

Reply:
Also, come to think of it, we should add some tests on this stuff...

Reply:
+1 for the second;
+1 for tests.

Reply:
I thought of `deltaSeconds` too, so will do that then. Let me do these changes then we can move to tests.